test/runtests.jl

```diff
@@ -0,0 +1,11 @@
+using Test
+using WinRPM
+using Libz, LibExpat
+
+#check that a file with a special char is downloaded correctly
+@testset "escaping" begin
+    pkgs = [WinRPM.select(WinRPM.lookup("libstdc++6", WinRPM.OS_ARCH), "libstdc++6"))]
+    WinRPM.do_install(WinRPM.Packages(pkgs))
```

Comment:
see https://github.com/JuliaPackaging/WinRPM.jl/pull/95#issuecomment-273035617 and https://github.com/JuliaPackaging/WinRPM.jl/pull/147/files/929e616ce9eac79c710d80640dba6946362064ca#diff-fce720c43af3c52c862fd7451c7374b8 - running tests should avoid making permanent observable modifications to a user's environment, such as installing a bunch of packages they may not need or want for anything other than verifying tests work. Better would be to do this in a temporary location that gets cleaned up afterwards.